javav2/usecases/creating_fsa_app/src/main/java/com/example/fsa/handlers/PollyHandler.java

```diff
@@ -0,0 +1,94 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.fsa.handlers;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.example.fsa.services.PollyService;
+import com.example.fsa.services.S3Service;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.Map;
+
+public class PollyHandler implements RequestHandler<Map<String, Object>, String> {
+    @Override
+    public String handleRequest(Map<String, Object> requestObject, Context context) {
+        S3Service s3Service = new S3Service();
+        PollyService pollyService = new PollyService();
+        String myValues = requestObject.toString();
```

Comment:
Variable name could be more descriptive.

Reply:
removed that one